src/main/kotlin/gg/essential/essential-loader.gradle.kts

```diff
@@ -0,0 +1,64 @@
+package gg.essential
+
+import gg.essential.gradle.multiversion.Platform
+import gg.essential.gradle.util.RelocationTransform.Companion.registerRelocationAttribute
+
+
+plugins {
+    id("gg.essential.loom")
+}
+
+val platform = Platform.of(project)
+
+val essentialLoader: Configuration by configurations.creating
+
+when {
+    platform.isLegacyForge -> {
+        dependencies {
+            "runtimeOnly"(essentialLoader("gg.essential:loader-launchwrapper:1.1.3")!!)
+        }
+        tasks.named<Jar>("jar") {
+            dependsOn(essentialLoader)
+            from({ zipTree(essentialLoader.singleFile) })
+        }
+    }
+
+    platform.isFabric -> {
+        dependencies {
+            "include"("modRuntimeOnly"("gg.essential:loader-fabric:1.0.0")!!)
+        }
+    }
+
+    platform.isModLauncher -> {
+        dependencies {
+            val relocatedPackage = findProperty("essential.loader.package")?.toString() ?: throw GradleException("""
+                No essential loader package set.
```

Comment:
The wording for this message also doesn't sound right, nor the property name.

Here's what I was thinking, although it could be changed slightly:

```kts
val relocatedPackage = findProperty("essential.loader.relocatedPackage") 
    ?: throw GradleException("""
        A package for the Essential loader to be relocated to has not been set.
        You need to set the `essential.loader.relocatedPackage` property in the project's `gradle.properties` file to a package where Essential's loader should be relocated to.

        For example: `essential.loader.relocatedPackage = "org.example.coolmod.relocated.essential"`
    """.trimIndent())

Reply:
I disagree, the property name describes what it is (the package of the essential loader) and the message gives enough information without being wordy.

Reply:
I'd agree on the message being insufficient without additional context. It also should **not** be the mod's package but an otherwise unused package nested below that (like in the proposed message) to avoid conflicts.
The property name seems fine as is. It is the package where you want your essential loader.